msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561081.java

```diff
@@ -99,7 +99,8 @@ public void handleUserInteraction() {
         authResult.assertSuccess();
 
         //extract the device id claim from the access token.
-        String deviceId = (String) JWTParserFactory.INSTANCE.getJwtParser().parseJWT(authResult.getAccessToken()).get("deviceid");
+        String deviceId =
+                (String) IDToken.parseJWT(authResult.getAccessToken()).get("deviceid");
```

Comment:
Is there no benefit in having these strings in a dedicated string resource file?

Reply:
Using a string resource file here is inapplicable as the AT is a generated value.